files/common/lib/udev/rules.d/60-dlpx-schedulers.rules

```diff
@@ -0,0 +1,5 @@
+#
+# Delphix: this setting disables the write throttle algorithm that is used in
+# the Linux IO scheduler as we've seen problems with it.
+#
+ACTION=="add|change", SUBSYSTEM=="block", KERNEL=="sd*", ATTR{queue/wbt_lat_usec}="0"
```

Comment:
I wonder if this set of matching rules could be inadvertently excluding some block devices that we care about. I think, Pavel, you're going to verify that and adjust if needed.

Reply:
Yeah, we probably need too do something like `KERNEL=="sd*|xvd*|virtio*'

Reply:
Yeah, I've tested this and it definitely doesn't apply to xvd/nvme/etc.
I'm considering removing KERNEL altogether, it seems to work. The downside is that it applies to other block devices as well, such as `loop` and `sr` and `zd`. It probably doesn't matter much for `loop` and `sr`, but I think for `zd` we'll probably want to exclude it, given the following behaviour:
```
$ cat /sys/block/zd0/queue/wbt_lat_usec
cat: /sys/block/zd0/queue/wbt_lat_usec: Invalid argument
$ sudo bash -c 'echo 0 >/sys/block/zd0/queue/wbt_lat_usec'
$ cat /sys/block/zd0/queue/wbt_lat_usec
0
```
I guess the safer way would be to selectively include, at the risk of missing new ones, rather than do a selective exclude and then accidentally mess with devices that we shouldn't be messing with.

Reply:
The impact of missing new ones would be rather huge (system-wide hang on a customer system at some point in the future requiring a patch release). What's the impact of killing wbt across the board? The kernel allows one to compile it out of the system altogether (via `CONFIG_BLK_WBT`), so I'm guessing this is not disastrous in theory.

Reply:
I don't know what's the impact of echoing `0` for zvols, which had `cat /sys/block/zd0/queue/wbt_lat_usec` return EINVAL. To be safe, I could set it across the board, but specifically exclude zvols.